test/verify/pytest_dynamic_loader.py

```diff
@@ -1,5 +1,4 @@
 import os
```

Comment:
AFAICS there is nothing substantial left in this file any more, is it? It could disappear completely.

Reply:
Removed.